resources/migrations/20200924062956-create-timers-and-time-spans.up.sql

```diff
@@ -0,0 +1,37 @@
+-- required to generate uuid in ID primary key fields
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
TODO: This migration step needs superuser rights. Fails when running from the command line via `lein -mf`. Find a fix.

Reply:
The fix is to create the uuid-ossp extension when the DB is provisioned, and not during migration.

The statement can remain in the sql file, to ensure migration fails if the extension is missing.